### ISSUE TITLE
Install?

### ISSUE THREAD

**Issue by CasperOng** [phhusson/vendor_hardware_overlay#625]
Hi, I am new.
I want to ask how to install a this file. I compiled it with github actions and get a apk file. I want to ask how can I install it?

**Comment by AnshhSingh** [comment]
just copy the apk into your phone's /system/overlay/, and set permission to rw-r--r--

**Comment by CasperOng** [comment]
> 

Thx. But I dont have enough space in my root directory. Can I extend it? my phone is using A/B partition btw

**Comment by ghost** [comment]
> Thx. But I dont have enough space in my root directory. Can I extend it? my phone is using A/B partition btw

If you're on vndklite variant, you can delete some files & regain space. You can remove these two folders `/system_ext/apex/com.android.vndk.{v28/v29}` which won't cause any damage.

**Comment by mapq48** [comment]
do i need root to do it?

**Comment by CasperOng** [comment]
> do i need root to do it?

Yes, you will need root to do it. Since it requires you to modify the "root" ( / ) of your phone.

**Comment by CasperOng** [comment]
It just like you editing files in C:/ in your Windows machine. You will need admin permission to do that.
